src/extensions/http_api/client.js

```diff
@@ -219,9 +219,9 @@ class HttpApiClient {
             const fullUri = `${baseUrl}${path}`
 
             if (this.body !== null) {
-                if (!['POST', 'PUT'].includes(method)) {
+                if (!['POST', 'PUT', 'PATCH'].includes(method)) {
```

Comment:
Add PATCH to verbsAcceptingBody at the top.

Reply:
We actually shouldn't need the rest of this block as it is done on line 242 below and using the verbsAcceptingBody array from above.